provisioning/osx/configure/src/services.spec.js

```diff
@@ -0,0 +1,398 @@
+import { expect } from 'chai';
+import { findWithPublishedPorts } from './services';
+
+describe('services', () => {
+  it('should find public services', () => {
+    const services = [
```

Comment:
This may be a little brittle for API versions other than the one you've tested against. Nothing to really do about it, just... FYI. It does change slightly between the dot versions.